demisto_sdk/commands/format/update_script.py

```diff
@@ -17,16 +20,52 @@ class ScriptYMLFormat(BaseUpdateYML):
             output (str): the desired file name to save the updated version of the YML to.
     """
 
-    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '', no_validate: bool = False):
+    def __init__(self, input: str = '', output: str = '', path: str = '', from_version: str = '', no_validate: bool = False,
+                 update_docker: bool = False):
         super().__init__(input, output, path, from_version, no_validate)
+        self.update_docker = update_docker
         if not from_version and self.data.get("type") == TYPE_PWSH:
             self.from_version = '5.5.0'
 
+    @staticmethod
+    def update_docker_image_in_script(script_obj: dict, from_version: Optional[str] = None):
```

Comment:
why making it a static function in scriptYML? I think that adding it to BaseUpdateYML is better as ScriptYMLFormat and IntegrationYMLFormat both inherent from it.

Reply:
It's mostly a matter of style regarding when to use static or not. In this case there are also other yml formatters that extend BaseUpdateYML, so it is less fitting.